apis/sgv2-graphqlapi/src/test/java/io/stargate/sgv2/graphql/integration/util/GraphqlClient.java

```diff
@@ -115,4 +128,29 @@ protected String urlEncode(String parameter) {
       throw new AssertionError(e);
     }
   }
+
+  protected String get(String authToken, String path, int expectedStatusCode) throws IOException {
+    Request request;
+    if (authToken != null) {
+      request =
+          new Request.Builder().url(path).get().addHeader("X-Cassandra-Token", authToken).build();
+    } else {
+      request = new Request.Builder().url(path).get().build();
+    }
+    final Response response;
+    try {
+      response = okHttpClient.newCall(request).execute();
+    } catch (SocketTimeoutException e) {
+      throw new IOException(
+          String.format(
+              "Timeout (%s) for GET request from URL '%s': service not running?",
+              request.url(), e.getClass().getName()));
+    }
+    assertThat(response.code()).isEqualTo(expectedStatusCode);
+
+    ResponseBody body = response.body();
+    assertThat(body).isNotNull();
+
+    return body.string();
+  }
```

Comment:
why not using rest assured here that's given as default test framework for HTTP? we could get rid of the OkHttp?

Reply:
Honestly out of laziness, adapting the existing code was quicker. But if RestAssured has a low-level querying API I can take a look.

Reply:
Done.